bin/pip_constraint_helpers.py

```
@@ -73,5 +73,5 @@ def run_cmd(cmd: list[str] | tuple[str, ...]) -> None:
     :param cmd: The command to invoke.
     """
     escaped_cmd = shlex.join(cmd)
-    print(f'Invoking the following command: {escaped_cmd!s}')  # noqa: T201, WPS421
+    print(f'Invoking the following command: {escaped_cmd!s}')  # noqa: T201
```

Ruff (0.16.1)

[warning] `noqa` comment used instead of `ruff: ignore`

Use `ruff: ignore` instead

(RUF105)